src/gravel/api/nfs.py

```diff
@@ -164,9 +174,10 @@ async def get_export_ls(service_id: str) -> List[int]:
     '/export/{service_id}/{export_id}',
     name='nfs export detail',
     response_model=NFSExportModel)
-async def get_export_info(service_id: str, export_id: int) -> NFSExportModel:
+async def get_export_info(request: Request, service_id: str, export_id: int) -> NFSExportModel:
```

Comment:
break this line?

Reply:
Maybe in a followup where I'm tempted to push everything through `black`. (Don't want to do it yet because this PR is too large).